browser/components/EditProject.tsx

```diff
@@ -110,33 +111,35 @@ export default class EditProject extends React.Component<Props, State> {
   }
 
   async componentDidUpdate(prevProps) {
-    const projectInfo = await this.getProject();
-    const projectGroups = [];
-    for (const group of projectInfo.groups) {
-      projectGroups.push(group.group_id);
-    }
-    const projectUsers = [];
-    let projectDetails = {
-      project_id: null,
-      project_license: null,
-      project_name: null,
-      project_url: null,
-    };
-    if (projectInfo) {
-      projectDetails = projectInfo.project;
-      for (const user of projectInfo.users) {
-        projectUsers.push(user.company_alias);
+    if (this.props !== prevProps) {
```

Comment:
Careful -- this doesn't do what you think it does. This will pretty much always be a true statement since these are references.

Reply:
I am using this to act as a debounce of sorts so it only runs when the selected project changes. I can pull this out if you want but I was following the pattern from EditGroup.

Reply:
After reading https://reactjs.org/docs/react-component.html#componentdidupdate I will be more specific on the triggering item in question.